server.py

```diff
@@ -66,6 +93,19 @@ def api_user(u):
     return json.dumps({})
 
 
+@app.route('/api/user/<u>/messages')
+def api_user_messages(u):
+    """ Gets an array of messages sent or received by a user """
+    return json.dumps([])
+
+
+@app.route('/api/user/<u>/messages/new')
```

Comment:
Only `GET` is allowed by default so this will need `@app.route('/api/user/<u>/messages/new', methods=['GET', 'POST'])